### PR TITLE
Translate nav into ja

### PR DESCRIPTION
**Pull request by bobstrange**
Part of #220

Navigation bar translation.

- Original
	- [`nav`](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//typescriptlang-org/src/copy/en/nav.ts?rgh-link-date=2020-02-06T15%3A46%3A01Z)
- Diff
	- https://github.com/microsoft/TypeScript-Website/commit/88539cfbebf444d4ee16360791db5cddd1bb2844

### CODE REVIEW

File: packages/typescriptlang-org/src/copy/ja/nav.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  nav_download: "ダウンロード",
+  nav_community: "コミュニティ",
+  nav_playground: "プレイグラウンド",
+  nav_playground_short: "プレイグラウンド",
+  nav_handbook: "ハンドブック",
+  nav_tools: "ツール",
+  nav_search_placeholder: "検索",
+  nav_search_aria: "TypeScript サイト内を検索",
+  nav_beta: "Search the TypeScript site",
+  nav_beta_notification:
```
Comment (bobstrange) [inline packages/typescriptlang-org/src/copy/ja/nav.ts]: @orta

It looks we no longer used both `nav_beta` and `nav_beta_notification` on the `TopNav`.
Can I remove them ?

https://github.com/microsoft/TypeScript-Website/blob/v2/packages/typescriptlang-org/src/components/layout/TopNav.tsx#L101-L103

Comment (orta) [reply packages/typescriptlang-org/src/copy/ja/nav.ts]: Yes!

Comment (bobstrange) [reply packages/typescriptlang-org/src/copy/ja/nav.ts]: @orta
Thank you :bow: I've opened a PR to remove them. ( #898 ).

**Comment by orta** [comment]
I'm going to intentionally close and re-open this PR, to see if https://github.com/orta/code-owner-self-merge works

**Comment by orta** [comment]
Looks like it only affects new PRs 👍🏻

**Comment by orta** [comment]
Let's get this in, I'd rather it get merged than stay idle for too long

**Pull request merged**